src/test/ui/const-fn-error.stderr

```diff
@@ -13,22 +13,20 @@ error[E0016]: blocks in constant functions are limited to items and tail express
    |                   ^
 
 error[E0015]: calls in constant functions are limited to constant functions, struct and enum constructors
-  --> $DIR/const-fn-error.rs:17:5
+  --> $DIR/const-fn-error.rs:17:14
    |
-17 | /     for i in 0..x { //~ ERROR calls in constant functions
-18 | |     //~| ERROR constant function contains unimplemented
-19 | |         sum += i;
-20 | |     }
-   | |_____^
+17 |     for i in 0..x { //~ ERROR calls in constant functions
+   |              ^^^^
+   |              |
+   |              in this macro invocation
```

Comment:
macro invocation? what macro? =)

Is this referring to the for loop desugaring?

Reply:
I believe it is the desugaring of the range. Is it ok if I tackle this on a follow up PR? The span is probably pointing at the `x`.

Reply:
Fixed.